tools/db/build/build_keyboards_script.php

```diff
@@ -200,17 +204,20 @@ function generate_keyboard_inserts() {
           {$this->sqlb($isRTL)},
           {$this->sqlb($isUnicode)},
           {$this->sqlb($isANSI)},
-    
+
           {$this->sqlb($includesWelcome)},
           {$this->sqlb($includesDocumentation)},
           {$this->sqlb($includesFonts)},
           {$this->sqlb($includesVisualKeyboard)},
-    
+
           {$this->sqlb($platform_windows)},
           {$this->sqlb($platform_macos)},
           {$this->sqlb($platform_ios)},
           {$this->sqlb($platform_android)},
           {$this->sqlb($platform_web)},
+          {$this->sqlb($platform_linux)},
+
+          0,
```

Comment:
Is this the deprecated flag?

Reply:
The deprecated flag gets set to 1 in the follow up script search-prepare-data.sql.